docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -82,7 +83,7 @@ The Data Center App Performance Toolkit framework is also set up for concurrency
 
 | Parameter | Recommended Value |
 | --------- | ----------------- |
-| Database instance class | [db.m4.large](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Summary) |
+| Database instance class | [db.m5.large](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Summary) |
```

Comment:
What are we making an instance size change? Does this effect the costs?

Reply:
AWS moved to a new generation of DB instances. So there is no way to select db.m4.large from the dropdown during quickstart wizard (only from CLI command).

Price: 
db.m4.large - $0.35 per hour for multi az deployment
db.m5.large - $0.342 per hour for multi az deployment

Confluence template was not updated yet, so it still uses db.m4.large.